Symfony/CS/Tests/Fixer/Contrib/NoUselessReturnFixerTest.php

```diff
@@ -34,6 +34,44 @@ public function provideFixCases()
         return array(
             array(
                 '<?php
+                    function bar($baz)
+                    {
+                        if ($baz)
+                            return $this->baz();
+                        else
```

Comment:
in the case that we have nothing to execute here, then we could strip out the `else` too?